server/src/subscription/model.rs

```diff
@@ -170,9 +170,25 @@ impl Subscription {
     }
 }
 
+pub async fn fetch_subscription_by_user_and_object<'a, E>(
+    user_id: i32,
+    object_id: i32,
+    executor: E,
+) -> Result<Option<Subscription>, sqlx::Error>
+where
+    E: Executor<'a>,
+{
+    let subscriptions = Subscriptions::fetch_by_object(object_id, executor).await?;
+    let subscription = subscriptions
+        .0
+        .into_iter()
+        .find(|subscription| subscription.user_id == user_id);
+    Ok(subscription)
+}
+
 #[cfg(test)]
 mod tests {
-    use super::{Subscription, Subscriptions};
+    use super::{fetch_subscription_by_user_and_object, Subscription, Subscriptions};
```

Comment:
Shouldn't it be possible to make `mod tests` public and import from that, instead? The disadvantage from your proposed change is that `fetch_subscription_by_user_and_object` is part of the production code.

Reply:
Good point. 
Since I've used it in another test, it is better to do as you say. 
Done in cae0f5b